modules/concertBidAdapter.js

```diff
@@ -57,8 +57,9 @@ export const spec = {
         slotType: bidRequest.params.slotType,
         adSlot: bidRequest.params.slot || bidRequest.adUnitCode,
         placementId: bidRequest.params.placementId || '',
-        site: bidRequest.params.site || bidderRequest.refererInfo.page
-      }
+        site: bidRequest.params.site || bidderRequest.refererInfo.page,
+        ref: bidderRequest.refererInfo.ref || window.document.referrer
```

Comment:
I'm curious about the fallback (because this choice is common enough that I wonder if it should be built into `refererInfo`). It would be used when prebid is running inside an isolated iframe, and its value would usually be the location of the parent frame - why is that preferable to `null`? Not knowing how this is used in the backend it looks like pollution of the data to me. (sometimes `ref` is the referrer, sometimes not, and you don't know which is which).

Reply:
@dgirardi You're right, that wasn't properly thought through; we don't want the `iframe`'s referrer. I see the pull request has been closed because of lack of CI, so I'll open another one with an update to the line above. Thanks for calling that out!

Reply:
Oh no, I just realised _I_ inadvertently closed this PR by messing around with `master` on our fork. 🤦 Sorry!